--- /tmp/stterm_repo/config.def.h	2018-03-02 13:33:09.343761426 -0500
+++ /tmp/stterm_repo/config.h	2018-03-02 13:37:34.706386397 -0500
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+char font[] = "Liberation Mono:pixelsize=24:antialias=true:autohint=true";
 int borderpx = 2;
 
 /*
@@ -49,7 +49,7 @@
  * blinking timeout (set to 0 to disable blinking) for the terminal blinking
  * attribute.
  */
-unsigned int blinktimeout = 800;
+unsigned int blinktimeout = 0;
 
 /*
  * thickness of underline and bar cursors
@@ -63,7 +63,7 @@
 static int bellvolume = 0;
 
 /* default TERM value */
-char termname[] = "st-256color";
+char termname[] = "xterm-256color";
 
 /*
  * spaces per tab
