--- config.h.def	2019-02-22 15:55:46.857022421 +0100
+++ config.h	2019-02-22 16:09:57.979833716 +0100
@@ -32,11 +32,11 @@
 };
 
 struct assoc assocs[] = {
-	{ "\\.(avi|mp4|mkv|mp3|ogg|flac|mov)$", "mpv" },
-	{ "\\.(png|jpg|gif)$", "sxiv" },
-	{ "\\.(html|svg)$", "firefox" },
-	{ "\\.pdf$", "mupdf" },
-	{ "\\.sh$", "sh" },
+	//{ "\\.(avi|mp4|mkv|mp3|ogg|flac|mov)$", "mpv" },
+	//{ "\\.(png|jpg|gif)$", "sxiv" },
+	//{ "\\.(html|svg)$", "firefox" },
+	//{ "\\.pdf$", "mupdf" },
+	//{ "\\.sh$", "sh" },
 	{ ".", "less" },
 };
 
@@ -92,8 +92,11 @@
 	{ CONTROL('L'),   SEL_REDRAW },
 	/* Run command */
 	{ 'z',            SEL_RUN, "top" },
-	{ '!',            SEL_RUN, "sh", "SHELL" },
+	//{ '!',            SEL_RUN, "sh", "SHELL" },
+	{ '!',            SEL_RUN, "st_background" },
 	/* Run command with argument */
-	{ 'e',            SEL_RUNARG, "vi", "EDITOR" },
+	//{ 'e',            SEL_RUNARG, "vi", "EDITOR" },
+	{ 'e',            SEL_RUNARG, "gvim", "EDITOR" },
 	{ 'p',            SEL_RUNARG, "less", "PAGER" },
+	{ 'o',            SEL_RUNARG, "mimeopenbg" },
 };
